<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Earthquake</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Desktop/CrossingWaves/icon.ico</normaloff>../../Desktop/CrossingWaves/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="buttonGet">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Data</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetResults">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>341</width>
      <height>311</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <column>
     <property name="text">
      <string>longitude</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>latitude</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>magnitude</string>
     </property>
    </column>
   </widget>
   <widget class="QRadioButton" name="radioButtonHour">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Past Hour</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonDay">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Past Day</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonWeek">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Past 7 Days</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonMonth">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Past 30 Days</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>547</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
